src/ManageCourses.Api/Mapping/CourseLoader.cs

```diff
@@ -67,10 +67,13 @@ public Course LoadCourse(IEnumerable<UcasCourse> courseRecords, IEnumerable<Ucas
             if (courseRecords.Count() > 0)
             {
                 // pick a reference course record for shared data such as the accrediting provider
-                // users can edit only running locations in UCAS so give preference to using running locations
+                // users can edit only:
+                // running locations in UCAS and
+                // published locations in UCAS
+                // so give preference to using running and published locations
                 // if there are any.
-                var organisationCourseRecord = 
-                    courseRecords.FirstOrDefault(x => x.Status != null && x.Status.ToLowerInvariant() == "r") 
+                var organisationCourseRecord =
+                    courseRecords.FirstOrDefault(x => x.Status != null && x.Status.ToLowerInvariant() == "r" && x.Publish != null && x.Publish.ToLowerInvariant() == "y")
```

Comment:
I've checked the data we have and this seems reasonable. There aren't actually any blanks but no harm protecting against them. :+1:

Reply:
thanks